sbi/analysis/plot.py

```diff
@@ -961,8 +982,8 @@ def _get_default_opts():
         # formatting points (scale, markers)
         "points_diag": {},
         "points_offdiag": {
-            "marker": ".",
-            "markersize": 20,
+            "marker": "+",
+            "markersize": 10,
```

Comment:
Why do you dislike the dot? I personally find the cross rather ugly in python. I'm fine with reducing the size.

Reply:
Interesting, I really prefer the cross over the dot because it seems more precise (it precisely conincides with the two vertical lines from the 1D marginals). 
Especially when using the default 2D histogram, the big blue dot often covers large parts of the posterior. 
But I am fine with using a smaller dot in a different color.

Reply:
changed it to a dot of size 10 and in different color ✅

Reply:
Thanks!